lib/publiccloud/azure.pm

```diff
@@ -76,8 +76,8 @@ sub find_img {
 
 sub get_storage_account_keys {
     my ($self, $storage_account) = @_;
-    my $output = script_output("az storage account keys list --resource-group "
-          . $self->resource_group . " --account-name " . $storage_account);
+    my $cmd = "az storage account keys list --resource-group " . $self->resource_group . " --account-name " . $storage_account;
+    my $output = script_output( "echo \"\n $cmd\" > /dev/$serialdev; " .$cmd . " 2>/dev/$serialdev");
```

Comment:
Why not just
```suggestion
    my $output = script_output($cmd . " 2> /dev/$serialdev");
```

Reply:
I added the echo so the serial console shows the commans itself as well, otherwise there would be only the error message without any context:
https://mordor.suse.cz/tests/4185/logfile?filename=serial0.txt
You would have the error message but no information where it comes from.